internal/connector/connector.go

```diff
@@ -136,11 +138,18 @@ func Run(ctx context.Context, options Options) error {
 	ctx, cancel := context.WithCancel(ctx)
 	defer cancel()
 
-	client := &api.Client{
-		Name:      "connector",
-		Version:   internal.Version,
-		URL:       u.String(),
-		AccessKey: accessKey,
+	promRegistry := metrics.NewRegistry(internal.FullVersion())
+	responseDuration := prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace: "serverhttp",
+		Name:      "response_duration_seconds",
```

Comment:
Not sure if this should be `request` or `response`

Reply:
I am on the fence too, maybe just more specific wording `response_to_connector_duration_seconds`

Reply:
I think this needs to be named from the perspective of the person operating the connector.

Something like `infra_api_request_seconds`.

Reply:
I don't think there's any reason to be that specific. The Prometheus instance scraping this metric will add context. The final metric as seen by an end user will look something like `server_http_response_duration_seconds{job="infra-connector"}`

Reply:
`server_http_response_duration` is very misleading I think. It's not the server response duration, it's the clients view of the request.  Someone using this metric with a SaaS connector is going to be very confused by this name.

Reply:
The metric is scraped from `job=infra-connector` and represents the job's view of the world. This metric measures the duration of a request to the server. It's using `response` as the inverse to `http_request_duration_seconds` which measure the duration of a request to the connector. 

This is also the inverse of the server's view of the world which is measured by `http_request_duration_seconds{job='infra-server'}`.

The labels are equally as important as the metric name. We could even omit `server` from the metric name because it's captured by the `host` label which describe what the client is talking to.

Reply:
The labels definitely help for sure. I think it would be good to omit server. I think the users reading this metric won't understand that `server` implies "infra api server". 

Are you thinking that if we add a metric for timing to the kube api server we'd use the same metric name, and someone would distinguish the two from the "host" label ?

Reply:
I agree. Initially I thought `server` will help clarify things but it's causing more confusion. 

> Are you thinking that if we add a metric for timing to the kube api server we'd use the same metric name, and someone would distinguish the two from the "host" label ?

That's exactly my thought. There's no reason to be specific about the metrci name since there's many things the connector needs to communicate with, Infra server being only one of them. Others may include the k8s API server or other integrations that happen to use HTTP

Reply:
Makes sense! I wonder if `http_client_request_duration` would work for this